### ISSUE TITLE
No such file or directory on supplied paths should fail (exit code != 0)

### ISSUE THREAD

**Issue by m-johansson** [ansible-community/ansible-lint#587]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version
ansible 2.8.4
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.7.4 (default, Aug 21 2019, 00:19:59) [GCC 8.3.0]
ansible-lint --version
ansible-lint 4.1.0
```

- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour

When ansible-lint fails to find a file specified by the user it should exit with a code != 0. I.e. start seeing it as  failed run if an supplied path does not point to a file or directory. 

# Actual Behaviour (Bug report only)

```
/work # ansible-lint test
WARNING: Couldn't open test - No such file or directory
/work # echo $?
0
/work # 
```

**Comment by berquist** [comment]
I work around this by doing
```bash
ansible-lint foo.yaml bar.yaml baz.yaml 2>&1 | tee ./.ansible-lint.tmp && test ! -s ./.ansible-lint.tmp
```

**Comment by ssbarnea** [comment]
Unable to reproduce on master, now invalid filename gets you an error.
